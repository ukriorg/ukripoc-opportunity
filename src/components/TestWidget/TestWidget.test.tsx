import React from "react";
import renderer from "react-test-renderer";
import { TestWidget } from "./index";

describe("TestWidget", () => (
    it("renders correctly", () => {
        const tree = renderer.create(<TestWidget />).toJSON();
        expect(tree).toMatchSnapshot();
    })
));

