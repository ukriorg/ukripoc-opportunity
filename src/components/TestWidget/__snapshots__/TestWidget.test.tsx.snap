// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestWidget renders correctly 1`] = `
<div>
  <div
    className="sc-bdVaJa cSBzFD"
  >
    <h1
      className="sc-bwzfXH cDtZmt"
    >
      Application complete
    </h1>
    <div
      className="sc-htpNat leRNHM"
    >
      Your reference number
      <br />
      <strong>
        HDJ2123F
      </strong>
      <p>
        <button
          className="sc-bxivhb hQDbuJ"
          disabled={false}
        >
          Hooray!
        </button>
      </p>
    </div>
  </div>
</div>
`;
